[FEATURE] Environnement pour Pix Concours (PIX-1514).

jonasgrilleres opened the pull request:
## :unicorn: Problème
Cette PR sert à préparer l'environnement pour Pix Concours. Le test aura lieu sur la RA.

## :robot: Solution
> _Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés._

## :rainbow: Remarques
> _Des infos supplémentaires, trucs et astuces ?_

## :100: Pour tester
> _Les instructions pour reproduire le problème, les profils de test, le parcours spécifique à utiliser, etc._

jonasgrilleres committed:
Do not display checkpoints on Pix Concours

jonasgrilleres committed:
Skip final chekpoint

Commit:
Hide levelup notif when pix concours

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr2098.review.pix.fr
- Orga: https://orga-pr2098.review.pix.fr
- Certif: https://certif-pr2098.review.pix.fr
- Admin: https://admin-pr2098.review.pix.fr
- API: https://api-pr2098.review.pix.fr/api/

 Please check it out!

jonasgrilleres committed:
Hide bottom presentation page

Commit:
Hide campaign didacticiel when Pix Concours

celineung committed:
[FEATURE] Pix Concours: Cacher le didacticiel (PIX-1516).

Merge pull request #2102 from 1024pix/pix-1516-pix-concours-hide-didacticiel

Commit:
Rename PIX_CONCOURS into PIX_CONTEST

jonasgrilleres committed:
Fetch Airtable test challenge when Pix Concours

celineung committed:
[FEATURE] Cacher le bas de la page présentation quand on démarre les campagnes Pix Concours (PIX-1563).

Merge pull request #2101 from 1024pix/pix-1563-hide-presentation-page-campaign

alexandrecoin committed:
Hide progress bar on pix contest challenge

alexandrecoin committed:
Hide feedback panel on challenge pages

Commit:
Add pix concours end page

pix-service-auto-merge committed:
[FEATURE] Retirer les liens "Signaler un problème" des différents challenges de Pix-Concours (PIX-1583).

alexandrecoin committed:
[FEATURE] Retirer la barre de progression durant les épreuves Pix-Concours (PIX-1582).

Merge pull request #2129 from 1024pix/pix-1582-hide-progress-bar-for-pix-contest-challenges

pix-service-auto-merge committed:
[FEATURE] Adapter l'algo des campagnes pour les tests Pix Concours (PIX-1573).

pix-service-auto-merge committed:
[FEATURE] Ajouter une page de fin aux campagnes de Pix Concours (PIX-1541).

alexandrecoin committed:
Update profile page main content for Pix-Contest

alexandrecoin committed:
Update navbar content for Pix-Contest

alexandrecoin committed:
Add example text and links for modified content

alexandrecoin committed:
Style modified content

alexandrecoin committed:
Update footer for Pix-contest

pix-service-auto-merge committed:
[FEATURE] Page d'accueil de Pix-concours (PIX-1556)

alexandrecoin committed:
Add migration for hasFinishedPixContest boolean on users

alexandrecoin committed:
Update user model with hasFinishedPixContest value

MelanieMEB committed:
Change order to get next challenge to not call dataFetcher

MelanieMEB committed:
Change title of challenge (remove /5)

Commit:
Update hasFinishedPixContest property

MelanieMEB committed:
Show resume bar only when assessment is not finished

alexandrecoin committed:
Add send Pix Contest results button on homepage

Commit:
Update user serializer

MelanieMEB committed:
[FEATURE] Pix concours : amélioration et gestion du bandeau.

Merge pull request #2159 from 1024pix/pix-contest-improving

Commit:
Display finished contest message

alexandrecoin committed:
Remove join campaign button when pix-contest is finished

pix-service-auto-merge committed:
[FEATURE] Ajout d'un bouton de finalisation de Pix-Concours (PIX-1596).

alexandrecoin committed:
Fix completed module wording

alexandrecoin committed:
Add description text under modules names

alexandrecoin committed:
Remove join campaign with code button

alexandrecoin committed:
Remove text on start campaign page

alexandrecoin committed:
Update homepage instructions

MelanieMEB committed:
Modify text

Commit:
Order translation files key

alexandrecoin committed:
[FEATURE] Correction de l'affichage Pix-Concours (PIX-1635).

Merge pull request #2173 from 1024pix/pix-1635-update-pix-contest-display

MelanieMEB committed:
Update text and link

MelanieMEB committed:
BUGFIX the display of the resume banner

- Do not show banner when the user has finished his Pix Contest
- Show the last not Finished Campaign Participation instead of not Shared

MelanieMEB committed:
[BUGFIX] [PIX-CONCOURS] Problèmes d'affichage du bandeau de reprise.

Merge pull request #2194 from 1024pix/bugfix-resume-banner

Commit:
Display username for Pix Contest

Commit:
Fix tests

pix-service-auto-merge committed:
[FEATURE] Pix Concours: Afficher le nom et prénom de l'utilisateur (PIX-1710).

MelanieMEB committed:
Show Progress Bar with only the index of challenges

alexandrecoin committed:
Improve instrutions wording

alexandrecoin committed:
[FEATURE] Amélioration des consignes de Pix-Concours (PIX-1728).

Merge pull request #2231 from 1024pix/pix-1728-improve-instructions-wording

MelanieMEB committed:
[FEATURE] [PIXCONCOURS] L'utilisateur doit savoir à quelle épreuve il est (PIX-1729).

Merge pull request #2227 from 1024pix/pix-1729-show-number-of-challenges

MelanieMEB committed:
[BUGFIX] Do not show resume banner when there is no campaignparticipation not finished

MelanieMEB committed:
Authorized to create user without validation

MelanieMEB committed:
Change text and color
